### PR TITLE
Modify script for new network config. [skip travis]

### PR DESCRIPTION
**Pull request by leeyr338**
Modify script for new network config:
- Do not need `ca` files;
- Do not need `id_card`;

### CODE REVIEW

File: scripts/create_cita_config.py [hunk 1]
```diff
@@ -400,39 +340,40 @@ def run_subcmd_create(args, work_dir):
...
+                                       title='subcommands',
+                                       help='additional help')
 
     #
     # Subcommand: create
     #
 
-    pcreate = subparsers.add_parser(
-        SUBCMD_CREATE, help='create a chain with nodes')
+    pcreate = subparsers.add_parser(SUBCMD_CREATE,
```
Comment (kaikai1024) [inline scripts/create_cita_config.py]: auto-fmt?pylint?

Comment (leeyr338) [reply scripts/create_cita_config.py]: I use command:
```shell
yapf -i create_cita_config.py
```
and yapf version is `0.27.0`.

Am I make something wrong?

Comment (kaikai1024) [reply scripts/create_cita_config.py]: I see.
Why use `yapf`? and it should be another commit if you format the code.

File: scripts/create_cita_config.py [hunk 2]
```diff
@@ -479,33 +421,31 @@ def parse_arguments():
...
-        type=NetworkAddressList.from_str_get_one,
-        metavar='IP:PORT',
-        help='Node network addresses for new nodes.')
+    pappend.add_argument('--node',
+                         required=True,
+                         type=NetworkAddressList.from_str_get_one,
+                         metavar='IP:PORT',
+                         help='Node network addresses for new nodes.')
 
     pappend.add_argument(
```
Comment (kaikai1024) [inline scripts/create_cita_config.py]: Why is this line not formatted?

File: scripts/create_cita_config.py (hunk 1 above)
Comment (leeyr338) [reply scripts/create_cita_config.py]: Oh, `pylint` is okay!

File: scripts/create_cita_config.py (hunk 2 above)
Comment (leeyr338) [reply scripts/create_cita_config.py]: I don't know why, maybe `yapf` makes something wrong. I will use `pylint`.

File: scripts/create_cita_config.py (hunk 1 above)
Comment (kaikai1024) [reply scripts/create_cita_config.py]: > For instance, code that conforms to the PEP 8 guidelines may not be reformatted. But it doesn't mean that the code looks good.

> It's based off of 'clang-format'

File: scripts/create_cita_config.py (hunk 2 above)
Comment (kaikai1024) [reply scripts/create_cita_config.py]: You can check it on next pr. use yapf or not

File: scripts/create_cita_config.py [hunk 3]
```diff
@@ -451,7 +390,7 @@ def parse_arguments():
         default=KeyKeyValueDict(),
         metavar='Contract.Argument=Value',
         help='Update constructor arguments for system contract.'
-        ' Can be specify more than once.')
+             ' Can be specify more than once.')
```
Comment (kaikai1024) [inline scripts/create_cita_config.py]: this

File: scripts/create_cita_config.py [hunk 4]
```diff
@@ -23,8 +23,8 @@ def update_search_paths(work_dir):
 
 def generate_prevhash(resource_dir):
     if resource_dir is not None \
-            and os.path.exists(resource_dir) \
-            and os.path.isdir(resource_dir):
+        and os.path.exists(resource_dir) \
```
Comment (kaikai1024) [inline scripts/create_cita_config.py]: no need to change

**Pull request merged**